### PR TITLE
Fix Flow 0.32.0 anotations

### PR DESCRIPTION
**Pull request by montogeek**
Implements https://github.com/MoOx/phenomic/pull/716

### CODE REVIEW

**Comment by thangngoc89** [comment]
Looks like there is some conflicts. Could you resolve it? And thanks for working on this

**Comment by montogeek** [comment]
Done :)

File: src/builder/webpack/config.common.js [hunk 1]
```diff
@@ -18,13 +18,10 @@ export default (config: PhenomicConfig): WebpackConfig => {
   const cacheDir = getCacheDir(config)
   const { webpackConfig = {} } = config
 
-  return {
+  const defaultConfig = {
     ...webpackConfig,
-    ...config.cache && hardSourceRecordsPath(cacheDir),
```
Comment (thangngoc89) [inline src/builder/webpack/config.common.js]: Flow doesn't like this?

File: src/loader-preset-default/index.js [hunk 2]
```diff
@@ -7,11 +7,10 @@ import initHeadPropertyFromContent
 import initBodyPropertyFromContent
   from "../loader-plugin-init-body-property-from-content"
 
-export default [
-   // $FlowFixMe Missing annotation wtf?
+const def: Array<mixed> = [
```
Comment (thangngoc89) [inline src/loader-preset-default/index.js]: Nice trick. I learnt something new today.

**Comment by thangngoc89** [comment]
@montogeek could you rebase this branch?

File: src/loader-preset-default/index.js (hunk 2 above)
Comment (montogeek) [inline src/loader-preset-default/index.js]: It is from the docs https://flowtype.org/docs/arrays.html#exporting-arrays

File: src/builder/webpack/config.common.js (hunk 1 above)
Comment (montogeek) [inline src/builder/webpack/config.common.js]: Nope :/

**Comment by thangngoc89** [comment]
I'm not a git export but looks like sonething when went wrong with your rebase, it's showing @MoOx commits and this PR

**Comment by montogeek** [comment]
Indeed, yep, something got wrong

**Comment by thangngoc89** [comment]
Yep. Just force push this branch.

**Comment by thangngoc89** [comment]
Man. iOS auto correction is awful. Sorry about the above comment

**Comment by MoOx** [comment]
Closing in favor or #776

**Comment by MoOx** [comment]
@montogeek what did you use to rebase?

**Comment by montogeek** [comment]
@MoOx Sorry for this mess :(

I used `git pull --rebase upstream origin` from my forked repo been `origin` my repo and `upstream` thir repo

**Comment by MoOx** [comment]
Don't be sorry, it's really nice from you to work on that!
pull --rebase sucks from what I can see.

Instead the next time do "git rebase upstream/origin" when you are on your branch ;)

**Comment by montogeek** [comment]
Thanks, I will!

**Pull request closed without merging**